src/components/shared/globalTypes.ts

```diff
@@ -153,31 +156,44 @@ export class Directory extends FileSystemObject {
   changeCurrentWorkingDirectory(
     currentWorkingDirectory: Directory,
     path: string
-  ): Directory | undefined {
```

Comment:
What????

Reply:
lol, yeah this is an issue I'll be addressing soon. should be null.